### PR TITLE
Added build script for method_source

### PR DESCRIPTION
**Pull request by vathsalashetty25**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: m/method_source/method_source_ubi_8.5.sh [hunk 1]
```diff
@@ -1,42 +1,74 @@
 #!/bin/bash -e
 # ----------------------------------------------------------------------------
 #
-# Package       : method_source
-# Version       : 0.9.2
```
Comment (seth-priya) [inline m/method_source/method_source_ubi_8.5.sh:5]: if this version has been tested , it should not be removed

File: m/method_source/method_source_ubi_8.5.sh [hunk 2]
```diff
@@ -1,42 +1,74 @@
...
-# Source repo   : https://github.com/banister/method_source
-# Tested on     : UBI: 8.5
-# Language      : Ruby
-# Travis-Check  : True
-# Script License: Apache License 2.0
-# Maintainer's  : Balavva Mirji <Balavva.Mirji@ibm.com>
```
Comment (seth-priya) [inline m/method_source/method_source_ubi_8.5.sh:11]: add to the maintener field, don't remove the name of current maintainer

File: m/method_source/method_source_ubi_8.5.sh [hunk 3]
```diff
@@ -1,42 +1,74 @@
...
+    echo "Tests failed "
+  else
+    echo "Build & unit tests Successful "
+  fi
+fi
 
-bundle install --verbose
-bundle exec rake
\ No newline at end of file
+#Observed one failure:syntax error  and is in parity with Intel
```
Comment (seth-priya) [inline m/method_source/method_source_ubi_8.5.sh:74]: Please specify the test failure details, syntax error seems like it needs to be looked at, does the build work for top-of-tree?

File: m/method_source/method_source_ubi_8.5.sh (hunk 2 above)
Comment (vathsalashetty25) [reply m/method_source/method_source_ubi_8.5.sh:11]: I and Balavva raised PR on the same day..there was a confict that i copied her script and added to my PR accidently and then i deleted it..her PR got merged..she worked on 0.9.2 ..where i have raised PR for top of tree

File: m/method_source/method_source_ubi_8.5.sh (hunk 3 above)
Comment (vathsalashetty25) [reply m/method_source/method_source_ubi_8.5.sh]: Added detailed error log

Comment (vathsalashetty25) [reply m/method_source/method_source_ubi_8.5.sh]: Travis check passed..build and test both passing,deleted the failure comment

**Comment by seth-priya** [comment]
Closing in favor of https://github.com/ppc64le/build-scripts/pull/1828

**Pull request closed without merging**